src/System.Management.Automation/DscSupport/JsonDscClassCache.cs

```diff
@@ -107,17 +107,17 @@ public static class DscClassCache
         {
             get
             {
-                if (t_classCache == null)
+                if (_classCache == null)
                 {
-                    t_classCache = new Dictionary<string, DscClassCacheEntry>(StringComparer.OrdinalIgnoreCase);
+                    _classCache = new Dictionary<string, DscClassCacheEntry>(StringComparer.OrdinalIgnoreCase);
                 }
 
-                return t_classCache;
+                return _classCache;
             }
         }
 
         [ThreadStatic]
-        private static Dictionary<string, DscClassCacheEntry> t_classCache;
+        private static Dictionary<string, DscClassCacheEntry> _classCache;
```

Comment:
I wonder why `t_` was removed as our named conversion states https://github.com/PowerShell/PowerShell/blob/master/docs/dev-process/coding-guidelines.md#naming-conventions

Reply:
Thank you; I went through the new code and updated members according to naming convention.